js/app.js

```diff
@@ -0,0 +1,55 @@
+"use strict";
+
+(function(){
+  angular
+  .module("wdiRadio", [
+    "ui.router",
+    "ng-token-auth",
```

Comment:
are you using `ng-token-auth` It's used for token based authentication but unless you are planning to add users or authentication, I would leave it off